packages/xo-server/src/xapi/mixins/vm.mjs

```diff
@@ -213,6 +213,10 @@ const methods = {
       )
     }
 
+    if (vm.platform.vtpm === 'true') {
```

Comment:
can vm.platform be undefined/null ?

Reply:
Good question: no this field existing since XenServer 4.0.